integration_tests/mockApis/educationAndWorkPlanApi.ts

```diff
@@ -320,24 +320,36 @@ const stubGetInduction = (options?: {
             options.hasQualifications === true
               ? [
                   {
+                    reference: '45b969cc-00c8-41c9-8a79-133d2ccf6326',
```

Comment:
The response from the API when retrieving an Induction now includes the reference and audit (who/when) fields for each qualification